spec/support/command_line_helpers.rb

```diff
@@ -6,4 +6,10 @@ def output_results(*)
       include(*)
     ).to_stdout
   end
+
+  def not_output_results(*)
```

Comment:
Could we use `define_negated_matcher` at file bottom here instead? See example https://github.com/mastodon/mastodon/blob/main/spec/support/matchers/http_link_header.rb#L33

Reply:
Done